### ISSUE TITLE
[dashboard] missing contrast of background to scrollbar

### ISSUE THREAD

**Issue by susnux** [nextcloud/server#58381]
### Summary

The dashboard has no content container thus the scrollbar is on the background and should have proper color

<img width="39" height="639" alt="Image" src="https://github.com/user-attachments/assets/bcd2d6a5-55b2-440c-9525-ec534edaa2b1" />
